src/bridge/RoomAccessSyncer.ts

```diff
@@ -265,6 +291,26 @@ export class RoomAccessSyncer {
         await botClient.sendStateEvent(roomId, "m.room.power_levels", plContent, "");
     }
 
+    public async onFailedMessage(req: BridgeRequest, server: IrcServer, channel: string) {
+        const key = `${server.getNetworkId()}:${channel}`;
+        const currThreshold = this.accessRefreshCount.get(key) || 0;
```

Comment:
`currThreshold` should be named `currRefreshCount`.
It's not a threshold as far as I can tell.